core/src/test/java/tech/pegasys/eth2signer/core/multikey/BlsArtifactSignerFactoryTest.java

```diff
@@ -74,15 +94,16 @@ void cleanup() {
 
   @Test
   void createsArtifactSignerFromKeyStoreUsingRelativePaths() {
-    final Path relativeKeystorePath = Path.of(KEYSTORE_FILE);
-    final Path relativePasswordPath = Path.of(PASSWORD_FILE);
+    final Path relativeKeystorePath = Path.of(KEYSTORE_FILE_NAME);
+    final Path relativePasswordPath = Path.of(PASSWORD_FILE_NAME);
     final ArtifactSigner artifactSigner =
         artifactSignerFactory.create(
             new FileKeyStoreMetadata(relativeKeystorePath, relativePasswordPath, KeyType.BLS));
 
     assertThat(relativeKeystorePath).isRelative();
     assertThat(relativePasswordPath).isRelative();
-    assertThat(artifactSigner.getIdentifier()).isEqualTo("0x" + PUBLIC_KEY);
+    assertThat(artifactSigner.getIdentifier())
+        .isEqualToIgnoringCase(blsKeyPair.getPublicKey().toString());
```

Comment:
before, this test explicitly indicated that the getIdentifier had an 0x prefix - this version, doesn't demonstrate that.
Having said that - I don't KNOW that this test is meant to prove that ... but it was nice to have it explicit _somewhere_ - rather than ensuring one API matches another.

Reply:
Is it worth adding an extra line to ensure:

assertThat(artifactSigner.getIdentifier()).startsWith("0x")?

Reply:
yeah, I can add that back.